Assets/scss/components/_home-container.scss

```diff
@@ -0,0 +1,129 @@
+.home-container {
+  background-image: url("../../images/hero-container-background.jpg");
```

Comment:
@urvi04s Please use image that is already in assets according to design and don't render image as background-image, render it as img tag or store the value of background-image value in a css variable in inline style then use it. So later on when we'll integrate into AEM then it'll be easy.
Cc:- @mathrik  @msharmaRP @jvaishnavrp

Reply:
Done thanks for the suggestion